tritonbench/operators/conv/__init__.py

```diff
@@ -0,0 +1,2 @@
+# (c) Meta Platforms, Inc. and affiliates. Confidential and proprietary.
```

Comment:
If this is an open-source operator, we shouldn't include this in the file header